products/apigateway/terraform.yaml

```diff
@@ -73,6 +79,11 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         default_from_api: true
       gatewayConfig: !ruby/object:Overrides::Terraform::PropertyOverride
         ignore_read: true
+      apiConfigId: !ruby/object:Overrides::Terraform::PropertyOverride
+        default_from_api: true
+    custom_code: !ruby/object:Provider::Terraform::CustomCode
+      extra_schema_entry: templates/terraform/extra_schema_entry/api_config.erb
```

Comment:
Are we unable to add this field using `virtual_fields`? That's how we'd normally add a field like this.

Reply:
No, I need the field to have ForceNew + ConflictsWith 

https://github.com/GoogleCloudPlatform/magic-modules/blob/cb6d34473a12c2ebceb51c9dac429392b2a1adbd/templates/terraform/resource.erb#L106